skimage/meson.build

```
@@ -51,14 +51,15 @@ use_pythran = run_command(py3,
   check: true
 ).stdout().strip() == '1'
 
-incdir_pythran = run_command(py3,
-  [
-    '-c',
-    'import os; os.chdir(".."); import pythran; print(os.path.dirname(pythran.__file__));'
-  ],
-  check: true
-).stdout().strip()
-inc_pythran = include_directories(incdir_pythran)
+if use_pythran:
```

```suggestion
if use_pythran
```
Hey @jstriebel, thanks for the PR. I'm not familiar with all the context around this but it seems like [the CI is unhappy about the trailing colon](https://github.com/scikit-image/scikit-image/actions/runs/3949144262/jobs/6759988544#step:4:197), see also https://mesonbuild.com/Syntax.html#if-statements.

Not sure whether we need a `== true` or similar. Does this work locally for you?

---

Hi @lagru, thanks for the hint! Tbh I didn't test this locally, but I'll try it after this change and will let you know.